### DISCUSSION TITLE
Need help with DNS records not propagating

### DISCUSSION THREAD

**Discussion by shibilyt** [vercel/community#2220] [Help]
### Summary

I am building a next.js app with Clerk.app as the auth provider. I'm trying to make the project go to production and have bought the domain via Vercel. Now, I've updated the CNAME records required for the Clerk in the vercel Dashboard. But the records are not being propagated. How do I fix this?
![image](https://user-images.githubusercontent.com/3939547/235051124-ded7d026-60cb-4f7f-a7d3-803408cf98ed.png)

**Comment by amyegan** [comment]
Hello @shibilyt. It is normal for changes to take up to 24-48 hours to fully propagate. You can check DNS propagation status using a tool like [whatsmydns.net](https://www.whatsmydns.net/) or `dig`

Our documentation and help guides have more info:
- [Domains Overview](https://vercel.com/docs/concepts/projects/domains#)
- [DNS propagation](https://vercel.com/docs/concepts/projects/domains#dns-propagation)
- [Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)
- [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)
